src/dwn.ts

```diff
@@ -68,7 +68,7 @@ export class Dwn {
       message = JSON.parse(requestString);
     } catch (error) {
       return new MessageReply({
-        status: { code: 400, detail: error.message }
+        status: { code: 400, detail: 'unable to JSON parse request' }
```

Comment:
Should the bytes to string be moved outside the try catch block?

Reply:
Not worth it at the moment, if it's not able to convert bytes to string, it's not able to JSON parse.